.github/workflows/ci.yml

```diff
@@ -198,23 +198,23 @@ jobs:
         shell: bash
 
   linux-build-core-sanitizers-coverage:
-    name: "Linux (gcc-5, Python 3.6): core build, sanitizers, coverage"
-    runs-on: ubuntu-18.04
+    name: "Linux (gcc-10, Python 3.9): core build, sanitizers, coverage"
+    runs-on: ubuntu-20.04
     env:
-      CC: gcc-5
-      CXX: g++-5
+      CC: gcc-10
+      CXX: g++-10
     steps:
       # gcc-5 has to be used with 18.04, because the integrated version from 16.04-image has a bug with linker flags:
```

Comment:
Only minor, but can you remove the comment? (Since we switch now to gcc-10 / 20.04)

Reply:
I removed it but in the C++14 PR, sorry.

Reply:
Ok :) Then we leave it as is. It would be good next time, to have it in one PR though.